packages/material-ui-lab/src/TreeView/TreeView.test.js

```diff
@@ -289,29 +350,29 @@ describe('<TreeView />', () => {
         </TreeView>,
       );
 
-      fireEvent.click(getByTestId('icon'));
+      fireEvent.click(screen.getByTestId('icon'));
 
       expect(handleNodeToggle.callCount).to.equal(0);
     });
   });
 
   describe('Accessibility', () => {
     it('(TreeView) should have the role `tree`', () => {
-      const { getByRole } = render(<TreeView />);
+      render(<TreeView />);
 
-      expect(getByRole('tree')).not.to.equal(null);
+      expect(screen.getByRole('tree')).not.to.equal(null);
     });
 
     it('(TreeView) should have the attribute `aria-multiselectable=false if using single select`', () => {
-      const { getByRole } = render(<TreeView />);
+      render(<TreeView />);
 
-      expect(getByRole('tree')).to.have.attribute('aria-multiselectable', 'false');
+      expect(screen.getByRole('tree')).to.have.attribute('aria-multiselectable', 'false');
     });
 
     it('(TreeView) should have the attribute `aria-multiselectable=true if using multi select`', () => {
-      const { getByRole } = render(<TreeView multiSelect />);
+      render(<TreeView multiSelect />);
 
-      expect(getByRole('tree')).to.have.attribute('aria-multiselectable', 'true');
+      expect(screen.getByRole('tree')).to.have.attribute('aria-multiselectable', 'true');
```

Comment:
Maybe we should have left this change for another pull request, it makes reviewing the change more challenging.

Reply:
Actually, did we settle on this approach? I have seen it mentioned in https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen. cc @mui-org/core-team.

Reply:
I've removed it from the PR to see if it makes it easier to review.